alacritty_terminal/src/grid/storage.rs

```diff
@@ -831,8 +836,15 @@ mod test {
             visible_lines: Line(0),
             len: 9,
         };
-        assert_eq!(storage.inner, shrinking_expected.inner);
-        assert_eq!(storage.zero, shrinking_expected.zero);
+        assert_eq!(
+            storage.inner[..shrinking_expected.zero],
+            shrinking_expected.inner[..shrinking_expected.zero]
+        );
+        assert_eq!(
+            storage.inner[storage.zero..],
+            shrinking_expected.inner[shrinking_expected.zero..]
+        );
```

Comment:
This only tests part of `inner`, for example in this test `storage.inner[..shrinking_expected.zero].len() == 5`, and there are more than 5 elements to check for. I'm not sure if there's a super elegant way to test from `zero` to `zero+len` without some way to wrap around.

Reply:
I've reverted a fix for this test at some point...

Reply:
Should be fine now @nixpulvis